phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/apps/ViewApps.java

```diff
@@ -26,6 +26,7 @@
 import javafx.event.EventHandler;
 import javafx.scene.input.MouseEvent;
 import me.xdrop.fuzzywuzzy.FuzzySearch;
+import org.eclipse.jgit.util.StringUtils;
```

Comment:
Can you change this import to the apache dependency? I didn't even know that jgit contains such a method...